api/db/services/file_service.py

```
@@ -155,47 +174,25 @@ def is_parent_folder_exist(cls, parent_id):
-    @DB.connection_context()
-    def get_kb_folder(cls, tenant_id):
+    def get_kb_folder(cls):
         for root in cls.model.select().where(
-                (cls.model.tenant_id == tenant_id), (cls.model.parent_id == cls.model.id)):
+                (cls.model.parent_id == cls.model.id)):
             for folder in cls.model.select().where(
-                    (cls.model.tenant_id == tenant_id), (cls.model.parent_id == root.id),
+                    (cls.model.parent_id == root.id),
                     (cls.model.name == KNOWLEDGEBASE_FOLDER_NAME)):
                 return folder.to_dict()
         assert False, "Can't find the KB folder. Database init error."
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)